flake-info/src/data/export.rs

```diff
@@ -315,7 +316,12 @@ impl From<import::NixOption> for Derivation {
                 PandocOption::LuaFilter(man_filter),
             ]);
 
-            let result = pandoc.execute().unwrap();
+            let result = pandoc.execute().or_else::<PandocError, _>(|err| {
+                error!("Pandoc couldnt parse documentation of '{}'", name);
+                error!("{}", err);
+                Ok(PandocOutput::ToBuffer(String::new()))
+            }).unwrap();
```

Comment:
```suggestion
            let result = pandoc.execute().expect(format!("Pandoc could not parse documentation of '{}'", name));
```

This would solve @erikarvstedt's remarks